lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3684,6 +3685,13 @@ namespace {
                                               /*throws*/ false,
                                               dc, decl);
 
+      if (auto *enumDecl = dyn_cast<EnumDecl>(dc)) {
+        if (dyn_cast_or_null<clang::NamespaceDecl>(enumDecl->getClangDecl())) {
+          result->setStatic();
+          result->setImportAsStaticMember();
```

Comment:
Instead of this, is it possible to go down the `importGlobalAsMethod` path above?